hdr_html_canvas_element.md

```diff
@@ -157,7 +189,7 @@ The component signals are mapped to red, green and blue tristimulus values accor
 
 #### rec2100-pq
 
-The component signals are mapped to red, green and blue tristimulus values according to the PQ system system specified in Rec. ITU-R BT.2100.
+The component signals are mapped to red, green and blue tristimulus values according to the Perceptual Quantizer (PQ) system system specified in Rec. ITU-R BT.2100.
```

Comment:
Since PQ is used earlier in the doc (including in a section header), this should appear earlier